packages/ui/src/components/w3m-modal-header/styles.ts

```diff
@@ -10,17 +10,20 @@ export default css`
     position: relative;
   }
 
-  .w3m-back-btn {
-    position: absolute;
-    left: 0;
+  .w3m-modal-header button {
     background-color: transparent;
     padding: 13px 17px;
-    margin-left: -17px;
     transition: opacity 0.2s ease-in-out;
   }
 
-  .w3m-back-btn:hover {
-    opacity: 0.6;
+  .w3m-modal-header button:hover {
+    opacity: 0.5;
+  }
+
+  .w3m-back-btn {
+    position: absolute;
+    left: 0;
+    margin-left: -17px;
```

Comment:
Replacing `-17px` with `-5%` results in a nearly identical look but it's a lot more clear and perhaps will be more responsive going forward.

Reply:
Most of ui elements in web3modal are fixed, including most paddings, so it is fine here. Main responsiveness in the modal comes from things like `display: flex` with `justify-content: space-between` for example.

Reply:
Got it